gradle/prCheck.gradle

```diff
@@ -63,48 +61,55 @@ def checkPull(changeId) {
         println "PR check will be skipped for Snyk security upgrades"
         return []
     }
-    def hexdevMatcher = title =~ /.*HEXDEV-[0-9]{4,5}.*/
-    def itaMatcher = title =~ /.*ITA-[0-9]{3,5}.*/
-    if (hexdevMatcher.matches() || itaMatcher.matches()) {
-        println "HEXDEV and ITA jiras are not validated"
-        return []
-    }
-    def pubdevMatcher = title =~ /.*(PUBDEV-[0-9]{4,5}).*/
-    if (!pubdevMatcher.matches()) {
-        return ["Pull Request ${changeId} doesn't have associated JIRA ticket. Please add jira id (PUBDEV-XXXX) in the title of the PR"]
-    }
-    def jiraId = pubdevMatcher[0][1]
-    def jiraDetails = jsonSlurper.parseText(new URL("https://h2oai.atlassian.net/rest/api/latest/issue/$jiraId").getText())
-    def status = jiraDetails.fields.status
-    if (status.name != 'In Progress') {
-        return ["Pull Request ${changeId} is linked to jira ticket https://h2oai.atlassian.net/browse/${jiraId}. " +
-                "This jira is not marked 'In Progress'. Please check if the associated jira ticket is correct and change the status to 'In Progress'"]
+    def issueMatcher = title =~ /.*GH-([0-9]{4,5}).*/
+    if (!issueMatcher.matches()) {
+        return ["Pull Request ${changeId} doesn't have associated GitHub issue. Please add the issue number (GH-XXXX) in the title of the PR"]
     }
-    def fixVersions = jiraDetails.fields.fixVersions
-    if (fixVersions.size() == 0) {
-        return ["Pull Request ${changeId} is linked to jira ticket https://h2oai.atlassian.net/browse/${jiraId}. " +
-                "This jira doesn't have a fix version set. Please check if the associated jira ticket is correct and validate with the team what should be the correct fixed version."]
+    def issueId = issueMatcher[0][1]
+    if (!pullDetails.body.contains("#${issueId}")) {
+        return ["The associated GitHub issue #${issueId} must be mentioned in the description of the PR."]
+    } 
+        
+    def issueDetails = jsonSlurper.parseText(new URL("https://api.github.com/repos/h2oai/h2o-3/issues/${issueId}").getText())
```

Comment:
I would slightly changed it to report both problems to developer at once. It happened to me that I only assign fix version and forgot to add also myself..

IMHO something like this could work:
```
    def errorMessage = []
    def issueDetails = jsonSlurper.parseText(new URL("https://api.github.com/repos/h2oai/h2o-3/issues/${issueId}").getText())
    if (issueDetails.milestone == null || issueDetails.milestone.state != "open"){
        errorMessage.add("The associated GitHub issue #${issueId} must be assigned to an opened milestone.")
    }
    if (issueDetails.assignee == null){
        errorMessage.add("The associated GitHub issue #${issueId} must  have an assignee.")
    }
    if (!errorMessage.isEmpty()) {
        return errorMessage
    }
```